src/adapter/src/command.rs

```diff
@@ -821,7 +854,7 @@ impl ExecuteResponse {
             }
             PlanKind::Subscribe => vec![Subscribing, CopyTo],
             StartTransaction => vec![StartedTransaction],
-            SideEffectingFunc => vec![SendingRows],
+            SideEffectingFunc => vec![SendingRows, SendingRowsImmediate],
```

Comment:
```suggestion
            SideEffectingFunc => vec![SendingRowsImmediate],
```

Same, should never involve clusterd.